src/client.rs

```diff
@@ -500,34 +515,89 @@ where
                 return Err(Error::Programming(err_msg.into()));
             }
 
+            /////////////////////////////////////////
+            // TUF-1.0.9 §5.1.7:
+            //
+            //     Persist root metadata. The client MUST write the file to non-volatile storage as
+            //     FILENAME.EXT (e.g. root.json).
+
+            self.store_metadata(&root_path, &MetadataVersion::None, &raw_signed_root)
+                .await;
+
+            // FIXME: This isn't a part of the spec, but we also store the versioned metadata. This
+            // allows us to initialize a repository from the local metadata.
```

Comment:
This comment is weird to have as a FIXME. If we're doing it for a reason, why would it need to be fixed?

Also, when you say "this allows us to..." can you say where that code is, to make it easier to understand what this is enabling?

Reply:
Sure, I'll switch to calling this a "NOTE". I used "FIXME" since I wanted to eventually get this updated in the spec (see https://github.com/theupdateframework/specification/issues/108).